sdk/java-sdk-spring/src/main/java/kalix/javasdk/annotations/TypeName.java

```diff
@@ -22,10 +22,10 @@
 import java.lang.annotation.Target;
 
 /**
- * Annotation to assign a logical type name to persisted events.
+ * Annotation to assign a logical type name to events.
  *
  * Kalix needs to identify each event in order to deliver them to the right event handlers.
- * If a logical type name isn't specified, Kalix will use the non-qualified class name.
+ * If a logical type name isn't specified, Kalix will use the full-qualified class name.
```

Comment:
Usually `fully qualified`. Could be with or without the hyphen.

```suggestion
 * If a logical type name isn't specified, Kalix will use the fully qualified class name.
```